hs_core/views/__init__.py

```diff
@@ -1176,7 +1176,7 @@ def my_resources(request, page):
 
     context = {
                 'collection': collection,
-                'numOwned': len(resource_collection)
+                'numOwned': resource_collection.count()
```

Comment:
is there a benefit to this, or do you just prefer using count?  I assume count is calculated in the db, vs len is via python, but haven't verified.

Reply:
'resource_collection' is now a QuerySet, so can't use 'len()' unless first convert it to a list and we don't want to convert it to a list.

Reply:
Yes 'count' is computed at the db.

Reply:
We no more creating this context variable.